### PR TITLE
Reduces attack log level of self-harm

### PR DESCRIPTION
**Pull request by TDSSS**
## What Does This PR Do
Moves attacks from mobs targetting themselves to the almost_all level of logging.

## Why It's Good For The Game
Because I don't care about the greytide in brig slowly punching himself to death

## Changelog
:cl:
tweak: lowered attack log level of self-harm
/:cl:

### CODE REVIEW

**Comment by JKnutson101** [comment]
Beep Boop. Remember, Self-Harm is not Crew-Harm, and doesn't conflict with Law 1. Beep Boop.

File: code/__HELPERS/mobs.dm [hunk 1]
```diff
@@ -298,7 +298,7 @@ This is always put in the attack log.
 			loglevel = ATKLOG_FEW
 		else if(istype(user) && !user.ckey && !target.ckey) // Attacks between NPCs are only shown to admins with ATKLOG_ALL
 			loglevel = ATKLOG_ALL
-		else if(!user.ckey || !target.ckey) // Player v NPC combat is de-prioritized.
+		else if(!user.ckey || !target.ckey || (user.ckey == target.ckey)) // Player v NPC combat is de-prioritized. Also no self-harm, nobody cares
```
Comment (farie82) [inline code/__HELPERS/mobs.dm]: If only it would also hide self harm for surrounding players. Oh how we can dream

Comment (Couls) [reply code/__HELPERS/mobs.dm]: did I hear make hearing self harm messages a preference toggle?

Comment (farie82) [reply code/__HELPERS/mobs.dm]: Oh baby you just know what I like to hear (and what not)

Comment (Couls) [reply code/__HELPERS/mobs.dm]: then I know you'll hate to hear that'll only be a thing when you code it 😉

**Pull request merged**